soc/nxp/imxrt/imxrt6xx/Kconfig.defconfig

```diff
@@ -115,3 +115,32 @@ choice USB_MCUX_CONTROLLER_TYPE
 endchoice
 
 endif # SOC_MIMXRT685S_CM33
+
+if SOC_MIMXRT685S_ADSP
```

Comment:
Might be worth attempting to de-centralize `Kconfig.defconfig` here (aka have a per-cluster Kconfig.deconfig placed in each cluster's directory and sourced in the main Kconfig.defconfig). Should make the main `Kconfig.defconfig` easier to read and, overall, should be cleaner. Maybe same could be applicable to `Kconfig` and `Kconfig.soc`? Although I question if it would be worth doing it for `Kconfig.soc` since it doesn't contain much?

Reply:
I think this is too much of an overkill right now and we could do that later when this files gets bigger. Also, imxrt595 already has this implementation and prefer to keep it consistent.